frontend/plan/components/schedule/Schedule.tsx

```diff
@@ -91,8 +118,8 @@ class Schedule extends Component {
                 "indigo",
             ];
             // some CIS120: `used` is a *closure* storing the colors currently in the schedule
-            let used = [];
-            return (c) => {
+            let used: string[] = [];
+            return (c: string) => {
```

Comment:
So if you look in `SchedBlock.tsx`, you'll see a `Color` enum or something to that effect. You should pull that enum out into the types file and then change all these strings and string arrays dealing with color into references to that enum. Should make it easier to add new colors later on, and ensures we don't have any typos!

Reply:
Moved `Color` enum into types, and then edited Block and Schedule to use the new `Color` type rather than the string.